app/tests/unit/specs/components/staking/DelegationsOverview.spec.js

```diff
@@ -70,6 +73,11 @@ describe(`DelegationsOverview`, () => {
 
     wrapper.setData({
       delegations,
+      balances: [{
+        total: 10000,
+        liquid: 50,
+        denom: "STAKE"
+      }]
```

Comment:
How did you fix the test actually? I still don't see it. Why was it empty Array? Not defined in data?